source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+.. _minio-site-replication-overview:
+
+=========================
+Site replication overview
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Site replication configures multiple independent MinIO sites or clusters as replicas.
+When configured this way, refer to the replica sites as either "peer sites" or "sites".
+
+Site replication assumes the use of an external identity provider (IDP).
+All configured deployments must use the same external IDP.
+
+
+What replicates across all sites?
+---------------------------------
+
+When enabling site replication on a set of sites, the following changes apply to the other sites:
```

Comment:
Try to restructure with either 'replication' or 'synchronization' as the key verb.

Spitballing:

```suggestion
Each MinIO deployment ("replica") in site replication synchronize the following actions:
```

Reply:
Also - server settings should also synchronize correct? Such as the majority of settings (excluding bucket notification targets) you can set via `mc admin confiig set` ?

Reply:
No, server settings do not synchronize. It's bucket, bucket settings, objects, object settings, and IAM stuff.

Reply:
interesting - server settings here also would include the AD/LDAP or OIDC settings you typically set via envvar.

I was hoping for a system where you could configure one deployment and all the others would 'inherit' those configuration settings (minus bucket notification and lifecycle tiering), but this is still something more than what is.